### PR TITLE
Refactor HelpCommand

### PR DESCRIPTION
**Pull request by zhengweii**

### CODE REVIEW

File: src/main/java/seedu/clinic/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -41,19 +41,22 @@
 
     public static final String COMMAND_WORD = "edit";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the supplier/warehouse"
-            + " identified by the index number used in the displayed supplier/warehouse list. "
-            + "Existing values will be overwritten by the input values.\n"
-            + "Parameters: ct/TYPE i/INDEX "
+    public static final String MESSAGE_USAGE = "Edits a supplier or warehouse at the specified index."
+            + " For suppliers, only its name, phone, email and remarks can be edited."
```
Comment (jeffreytjs) [inline src/main/java/seedu/clinic/logic/commands/EditCommand.java:45]: Should I include this in AddCommand as well? Since it's the first touch for users

**Review by jeffreytjs — COMMENTED** [review]
LGTM, just wondering if I should add the specification like those in View and Edit to Add.

File: src/main/java/seedu/clinic/logic/commands/EditCommand.java (hunk 1 above)
Comment (zhengweii) [reply src/main/java/seedu/clinic/logic/commands/EditCommand.java:45]: Ahh I might change to to something like the one in DeleteCommand!

**Review by tohyuting — COMMENTED** [review]
Looks good, except for the comment pointed out by Jeffrey! 👍

**Review by qlchan24 — APPROVED** [review]
LGTM

**Review by jeffreytjs — APPROVED** [review]
Really good work with cleaning up the code and standardising across all features

**Pull request merged**